bsp/esp32_s3_korvo_1/include/bsp/esp32_s3_korvo_1.h

```diff
@@ -13,18 +13,50 @@
 #pragma once
 
 #include "iot_button.h"
+#include "esp_codec_dev.h"
 #include "led_indicator.h"
+#include "driver/sdmmc_host.h"
+
+#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 0)
+#include "driver/i2s.h"
+#else
+#include "driver/i2s_std.h"
+#include "driver/i2s_tdm.h"
+#endif
 
```

Comment:
Could you also please add capabilities defines? Like this: https://github.com/espressif/esp-bsp/blob/master/bsp/esp-box-3/include/bsp/esp-box-3.h#L30

Reply:
Thanks, could you add _all_ the capabilities macros? And define them as '0' in they are not supported? https://github.com/espressif/esp-bsp/blob/master/bsp/esp32_s3_usb_otg/include/bsp/esp32_s3_usb_otg.h#L25